easybuild/easyconfigs/s/Schrodinger/Schrodinger-2021-4.eb

```
@@ -0,0 +1,31 @@
+easyblock='PackedBinary'
```

@mustafabar Welcome to EasyBuild, and thanks for making a contribution.

The test suite is complaining, because you need to use spaces around `=` here.
```suggestion
easyblock = 'PackedBinary'
```